tox.ini

```diff
@@ -38,41 +36,32 @@ sitepackages = false
 deps =
     flake8>=3.6.0,<4
     yamllint>=1.11.1,<2
-extras =
 skip_install = true
 usedevelop = false
-
-[testenv:doc]
-# doc requires py3 due to use of f'' strings and using only python3 as
-# basepython risks using python3.4 which is not supported.
-basepython = python3.7
-passenv = *
 commands =
-    python setup.py build_sphinx -n -W --builder=html
-extras =
-    docs
+    flake8 piedpiper-noop-function
+    bash -ec "yamllint $(find . -path ./template -prune -o -path ./tests -prune \
+        -o -name '*.yml' -print)"
 
-[testenv:build-docker]
-# skip under Windows
-platform = ^darwin|^linux
-# `usedevelop = True` overrided `skip_install` instruction, it's unwanted
-usedevelop = False
-# don't install Molecule in this env
-skip_install = True
+[testenv:sanity]
 deps =
-    setuptools_scm
-# reset pre-commands
+    pluggy==0.12.0
+    pytest==4.5.0
+    PyYAML==5.1
+skip_install = true
+usedevelop = false
 commands_pre =
-# build the docker container
+    wget https://github.com/AFCYBER-DREAM/piedpiper-mindflayer/archive/v1.0.0.zip \
+        -O /tmp/piedpiper-mindflayer.zip
+    unzip /tmp/piedpiper-mindflater.zip
 commands =
-    sh -c '\
-    TAG=$(python -c "from setuptools_scm import get_version; print(get_version())" | tr '+' '-') && \
-    echo Building version "$TAG"... && \
-	faas build --shrinkwrap && \
-	cd build/piedpiper-flake8-function && \
-    sudo docker build \
-        --pull --rm \
-        {posargs:-t quay.io/afcyber-dream/piedpiper-flake8-faas:$TAG} .\
-    '
-whitelist_externals =
-    sh
+    pytest ./piedpiper-mindflayer-1.0.0/tests/test_stack_yaml.py
+commands_post =
+    rm /tmp/piedpiper-mindflayer.zip
+
+[testenv:functional]
```

Comment:
Let's just remove this environment entirely for now

Reply:
See previous comment.